docs/running-documentation-locally.md

```diff
@@ -0,0 +1,45 @@
+﻿---
+title: Running the documentation locally
+category: Compiler Internals
+categoryindex: 200
+index: 999
+---
+# Running the documentation locally
+
+The source of this documentation website is hosted on https://github.com/fsharp/fsharp-compiler-docs.  
+
+## Setup
+
+`fsharp/fsharp-compiler-docs` will clone the `dotnet/fsharp` repository first to generate the documentation.  
+You can however, easily run the documentation locally and modify the `docs` from `dotnet/fsharp`.
```

Comment:
Maybe worth adding a sentence on why someone might need this.

Reply:
Isn't that a little obvious? You would like to locally how your changes will reflect the documentation website.

Reply:
👀 

Well personally I would be probably motivated to do this only when doing big changes on that front. 
But as you wish, feel free to keep it as it is if you will.